services/director-v2/src/simcore_service_director_v2/core/settings.py

```diff
@@ -272,23 +272,35 @@ class PGSettings(PostgresSettings):
     )
 
 
-class DaskSchedulerSettings(BaseCustomSettings):
+class DaskComputationalBackendSettings(BaseCustomSettings):
```

Comment:
The convention is that the fields are prefixed with the name of the Settings class except for the main 
application class that takes the name of the service (i.e. DIRECTOR_V2_).

Here the name of the setting is pretty long already so a shortcut might be acceptable

If you trully want to have these names for the cluster I suggest that you add them
as fields in the ``AppSettings``, otherwise, here they should be prefix ``DASKCOMPUTATIONALBACKEND_`` (perhaps ``DASKCB_``?)
but not ``DIRECTOR_V2_``

The settings in this file are all a bit wild regarding the conventions ... it will make it more difficult to have a plugin-based approach
for this service :-(

Reply:
ok renamed all the settings in there